app/views/poems/_form.html.erb

```diff
@@ -1,13 +1,8 @@
 <%= simple_form_for(@poem) do |form| %>
   <%= render 'errors', object: @poem %>
-
- u
       
   <%= form.input :title %>
-  <%= 
-    form.select :status,
-    ["Available", "Pending", "Accepted – Forthcoming", "Published"]
-  %>
+  <%= form.select :status, Poem::POEM_STATUSES %>
```

Comment:
not sure if it's kosher to use model constants in the view instead of wrapping them in a helper. Writing that makes me think not because it seems more tightly coupled, but I'm just responding to my gut feeling on it.